_posts/2018-07-20-totw-153.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Tip of the Week #153: Don't use using-directives"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/153
+type: markdown
+order: "153"
+---
+
+Originally posted as TotW #153 on July 17, 2018
+
+*by Roman Perepelitsa
+[(roman.perepelitsa@gmail.com)](mailto:roman.perepelitsa@gmail.com) and Ashley
+Hedberg [(ahedberg@google.com)](mailto:ahedberg@google.com)*
+
+**I view using-directives as time-bombs, both for the parties that deal in them
+and the language system.** -- *Ashley Hedberg* with apologies to Warren Buffett
+
+## tl;dr
+
+Using-directives (`using namespace foo`) are banned by the [Google style
```

Comment:
"Banned by the Google style guide probably isn't the most salient point for most people reading these tips from oss-land.  Maybe something more like "Using-directives are dangerous enough to be banned by the google style guide" which puts more emphasis on teh "dangerous" and less on the "google style guide"?